code/atelier style.css

```diff
@@ -0,0 +1,203 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
```

Comment:
Small thing - but we should name our files without spaces - so something like `atelier-style.css` would work fine